components/raftstore-v2/src/raft/storage.rs

```diff
@@ -58,8 +58,8 @@ pub struct Storage<ER> {
 impl<ER: RaftEngine> Storage<ER> {
     /// Creates a new storage.
     ///
-    /// All metadata should be initialized before calling this method. If the region is destroyed
-    /// `None` will be returned.
+    /// All metadata should be initialized before calling this method. If the
```

Comment:
Why the change?

Reply:
80c limit.

Reply:
In Rust, it's 99.

Reply:
I don't think so. Rust fmt default configures it as 80: https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#comment_width, and rust source code uses 80 too.

Reply:
Got it, never expect length of comment line is different from source code.

Reply:
Turns out I'm not the only one has the confusion: https://github.com/rust-lang/rustfmt/issues/3349#issuecomment-622522913

Reply:
Having looked closer, apparently rust source code uses different comment width as well.. Personally I think 80c comment is perfect, whilst source code is easier to digest because they are structured.

Reply:
Let's limit the comment width automatically by rustfmt, then it's no need to discuss it again and again. https://github.com/tikv/tikv/pull/13139 some comments should be adjusted after auto wrap, will work on fixing it once we get an agreement on the comment width.